deps/src/polymake_jlcxx.h

```diff
@@ -0,0 +1,23 @@
+#ifndef POLYMAKE_JLCXX
+#define POLYMAKE_JLCXX
+
+// This must be the very first include
+
+// workaround xcode 11.4 issue until upstream fix arrives in xcode:
```

Comment:
Super nitpick, but it should be either "work around" (verb) or "workaround for" (noun).

Reply:
applied